### PR TITLE
Prevent admins from pressing buttons that cause the server to reach out to external services and fail.

### PR DESCRIPTION
**Pull request by nevimer**
AAAAAA.

...

AHHHHHHHHH.

Locks up the server waiting for responses it will never get.

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
I need this

**Comment by Cyprex** [comment]
Why the heck does this crash instead of just silently timing out asynchronously and failing.

**Comment by Cyprex** [comment]
To be clear if the server locks up for a few seconds then that's not great but imo acceptable- admins can figure out NOT to press funny buttons.

We use CC lockups, and these really shouldn't cause issues...

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1456,6 +1456,9 @@
 		usr << browse(dat.Join("<br>"), "window=related_[C];size=420x300")
 
 	else if(href_list["centcomlookup"])
+		if(href_list) // BUBBER EDIT BEGIN - Crash Bandaid
```
Comment (Cyprex) [inline code/modules/admin/topic.dm:1459]: Also, href_list is guaranteed to have the key "centcomlookup", does that not ensure this condition is always true? I don't know how truthiness works with regards to lists.

Comment (nevimer) [reply code/modules/admin/topic.dm:1459]: > Also, href_list is guaranteed to have the key "centcomlookup", does that not ensure this condition is always true? I don't know how truthiness works with regards to lists.

If anything is in the contents of a variable its true. Theres some weird null differences.

Comment (nevimer) [reply code/modules/admin/topic.dm:1459]: Also it's intentionally like this as it doesn't make the linter cry like just adding a return does.

**Review by magatsuchi — APPROVED** [review]
approved, for the reasons listed in maintainerbus

make it priority that we get this entire subsystem either implemented or removed

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (magatsuchi) [reply code/modules/admin/topic.dm:1459]: nonissue bc of linter context

**Pull request merged**